### DISCUSSION TITLE
Add songs to playlist issue 20240719

### DISCUSSION THREAD

**Discussion by RamblerKevin** [rern/rAudio#1736] [Bug Reports]
I updated recently to 20240719 and since then the software stalls out when I click a directory to add and replace al tracks in the playlist. Using a Pi4. Anyone else having this issue? Happens on my larger USB 1TB drive but also my 64g USB thumb drive so size of the USB directory doesn't seem to be an issue. Thanks.

**Comment by Lubos-V** [comment]
as discussed - maybe linked?
#1706
